Modify the ts4900 dts as per an explicit customer requirement.

Upstream-Status: Inappropriate [configuration specific]

Signed-off-by: Florin Sarbu <florin@resin.io>

diff --git a/arch/arm/boot/dts/imx6qdl-ts4900.dtsi b/arch/arm/boot/dts/imx6qdl-ts4900.dtsi
index 99dc97e..9adeddc 100644
--- a/arch/arm/boot/dts/imx6qdl-ts4900.dtsi
+++ b/arch/arm/boot/dts/imx6qdl-ts4900.dtsi
@@ -377,7 +377,7 @@
 				MX6QDL_PAD_EIM_A17__GPIO2_IO21         0x1b0b1 // OFF_BD_RESET#
 				MX6QDL_PAD_EIM_A16__GPIO2_IO22         0x1b0b1 // EN_USB_5V#
 				MX6QDL_PAD_EIM_A19__GPIO2_IO19         0x1b0b1 // EN_LCD_3.3V
-				MX6QDL_PAD_GPIO_0__CCM_CLKO1           0x130b0 // Audio CLK
+				MX6QDL_PAD_GPIO_0__GPIO1_IO00          0x80000000
 				
 				MX6QDL_PAD_GPIO_5__GPIO1_IO05          0x1b0b1 // DIO_1
 				MX6QDL_PAD_GPIO_9__GPIO1_IO09          0x1b0b1 // DIO_3
